core/src/data_sources/data_source.rs

```diff
@@ -941,7 +941,6 @@ impl DataSource {
                 payload.insert("timestamp", document.timestamp as i64);
                 payload.insert("chunk_offset", c.offset as i64);
                 payload.insert("chunk_hash", c.hash.clone());
-                payload.insert("data_source_id", self.data_source_id.clone());
```

Comment:
Why don't we want to keep the `data_source_id` in Qdrant?

Reply:
First it serves no purpose. Second if we wanted to keep it we would have to migrate it which is very challenging

Reply:
For sure we will need it in the future to apply proper data segregation!

Reply:
This is the purpose `data_source_internal_id` as per the slack thread.